central/processbaseline/service/service_impl.go

```diff
@@ -27,19 +27,11 @@ var (
 	authorizer = perrpc.FromMap(map[authz.Authorizer][]string{
 		user.With(permissions.View(resources.ProcessWhitelist)): {
 			"/v1.ProcessBaselineService/GetProcessBaseline",
-
-			// TODO(ROX-6194): Remove after the deprecation cycle started with the 55.0 release.
-			"/v1.ProcessWhitelistService/GetProcessWhitelist",
 		},
 		user.With(permissions.Modify(resources.ProcessWhitelist)): {
 			"/v1.ProcessBaselineService/UpdateProcessBaselines",
 			"/v1.ProcessBaselineService/LockProcessBaselines",
 			"/v1.ProcessBaselineService/DeleteProcessBaselines",
-
-			// TODO(ROX-6194): Remove after the deprecation cycle started with the 55.0 release.
-			"/v1.ProcessWhitelistService/UpdateProcessWhitelists",
-			"/v1.ProcessWhitelistService/LockProcessWhitelists",
-			"/v1.ProcessWhitelistService/DeleteProcessWhitelists",
```

Comment:
Why is this diff here?

Reply:
The diff I pointed out above is for process baseline APIs. Why are we removing it in policy PR?

Reply:
I'm not sure I understand. The functions removed are for Process **Whitelist** service which has been deprecated since 55. Process **Baseline** are still around

Reply:
If you're question is why was this combined with a PR that touches policy fields.... good question. Unfortunately that ship sailed. Going forward something like this should not have been combined in the first place